src/main/scala/ReleaseExtra.scala

```diff
@@ -225,7 +225,7 @@ object ReleaseStateTransformations {
   }
 
   lazy val publishArtifacts = ReleaseStep(
-    action = publishArtifactsAction,
+    action = runPublishArtifactsAction,
```

Comment:
@jroper My preference would be to prioritize 0.8.[3,4] => 0.8.6 being backwards compatible rather than 0.8.5 => 0.8.6 since it would have been preferable to be backwards compatible in 0.8.5.